packages/account/src/Components/currency-selector/__tests__/currency-selector.spec.tsx

```diff
@@ -273,8 +273,8 @@ describe('<CurrencySelector/>', () => {
                     is_virtual: 1,
                     landing_company_shortcode: 'svg',
                     trading: {},
-                    token: 'a1-sLGGrhfYPkeEprxEop2T591cLKbuN',
-                    email: 'test+qw@deriv.com',
+                    token: '',
+                    email: '',
```

Comment:
Are these part of this PR?

Reply:
@amina-deriv there were no changes to currency selector. could you please try pulling latest master?